spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -17,16 +17,70 @@
 package com.intel.analytics.bigdl.utils
 
 import java.io._
-import java.nio.file.{Files, Paths}
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FSDataInputStream, FSDataOutputStream, FileSystem, Path}
 import org.apache.hadoop.io.IOUtils
 
+import scala.collection.mutable.ArrayBuffer
+
+/**
+ * A uniform approach to get Input/Output stream in BigDL.
+ * @param fileName
+ */
+private[bigdl] class File(fileName: String) {
+  private val inputStreams: ArrayBuffer[InputStream] = new ArrayBuffer[InputStream]()
```

Comment:
do we have the scenario we need to maintain multiple input stream for the same file ?